.circleci/config.yml

```diff
@@ -140,6 +136,59 @@ jobs:
       - store_test_results:
           path: /tmp/logs
 
+  gcp_test:
```

Comment:
NIT: add comment that the GCP tests are intentionally in a separate Docker based build, and add a link to this very PR. That way, future maintainers won't be tempted to change it back.

Reply:
Added a comment block.